ga4gh/frontend.py

```diff
@@ -261,6 +290,16 @@ def assertCorrectVersion(version):
         raise exceptions.VersionNotSupportedException()
 
 
+def authenticated(handler):
```

Comment:
it would be good to add a `functools.wraps` call somewhere here for easier debugging (I've tried to use that construct on all decorators I've added so far... see other examples in the code base)